saas_app/training_request/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+from django.contrib.auth.models import User
+from user.models import Users
+from internal_ops.models import FundCenter
+from submit_request.models import Currency
+
+
+# DB Model for the course
+class Course(models.Model):
+    title = models.CharField(max_length=100)
+    description = models.CharField(max_length=100)
```

Comment:
The description could get long for some courses. Can we increase the max length?

Reply:
Thanks for noticing Wanpeng! I have changed now the length to 2000.